frontend/src/main/components/Commons/Profits.js

```diff
@@ -2,23 +2,38 @@ import React from "react";
 import { Card } from "react-bootstrap";
 import ProfitsTable from "main/components/Commons/ProfitsTable"
 
-const dummyData = [
-    { id: 1, profit: 10, date: "2021-03-05" },
-    { id: 2, profit: 11, date: "2021-03-06" },
-    { id: 3, profit: 10, date: "2021-03-07" },
-    { id: 4, profit: 8, date: "2021-03-08" }
-];
+// const dummyData = [
+//     { id: 1, profit: 10, date: "2021-03-05" },
+//     { id: 2, profit: 11, date: "2021-03-06" },
+//     { id: 3, profit: 10, date: "2021-03-07" },
+//     { id: 4, profit: 8, date: "2021-03-08" }
+// ];
```

Comment:
Can we remove unused code instead of commenting it out?

Reply:
Removed.